packages/integration-tests/test/inFlightExitChallengeResponse.js

```diff
@@ -115,16 +135,14 @@ describe('In-flight Exit Challenge Response tests', async () => {
       console.log(`Bob called RootChain.startInFlightExit(): txhash = ${receipt.transactionHash}`)
 
       // Keep track of how much Bob spends on gas
-      const bobSpentOnGas = await rcHelper.spentOnGas(web3, receipt)
-
-      // Decode the transaction to get the index of Bob's output
-      const decodedTx = transaction.decodeTxBytes(bobTx)
-      const outputIndex = decodedTx.outputs.findIndex(e => e.owner === bobAccount.address)
+      bobSpentOnGas.iadd(await rcHelper.spentOnGas(web3, receipt))    
+      const outputIndex = decodedTx.outputs.findIndex(e => e.outputGuard === bobAccount.address)
```

Comment:
does this need some `.toLowerCase()` just in case 😝

Reply:
it is ETH so should be fine lol